ui/app/AppLayouts/Profile/stores/AdvancedStore.qml

```diff
@@ -25,6 +25,7 @@ QtObject {
                             root.fleet === Constants.status_prod
 
     readonly property bool isFakeLoadingScreenEnabled: localAppSettings.fakeLoadingScreenEnabled ?? false
+    property bool isManageCommunityOnTestModeEnabled: false
```

Comment:
Is this setting persisted somehow, or did you forget to push the respective NIM files? :)

Reply:
Alright, makes sense if it's a single shot thing...

Reply:
Yeah! Just the basic stuff needed without saving status!